akka-stream/src/main/scala/akka/stream/impl/StreamLayout.scala

```diff
@@ -251,7 +252,7 @@ private[akka] object StreamLayout {
          * materialized value computation of its submodules.
          */
         Atomic(this),
-        Attributes.none)
+        DefaultAttributes.nested)
```

Comment:
Okay. I see now. I have reverted this to `Attributes.none`. It also means that this together with changes in [here](https://github.com/akka/akka/commit/162bd9b8e365b5e545bc9f555e079efb611cf20c#diff-c4bb0aec5f8c2431f9bf843216660886L187) produces lots of more actor names with "uknown-operation" in its names. Lets merge this PR as it morphed into rename `Source.from` to `Source.tick` and fix remaining "unknown-operation" in other ticket.